### ISSUE TITLE
Testlink fail to set APACHE_HTTP_PORT_NUMBER & APACHE_HTTPS_PORT_NUMBER

### ISSUE THREAD

**Issue by Jacoon** [bitnami/bitnami-docker-testlink#93]
Hi,

we are using the bitnami/testlink:1.9.19 image. A few days ago we noticed that our testlink is not reachable anymore by our nginx. After some time of research I noticed that the testlink is not running on the configured ports 8080 and 8143 but on the default ports 80 and 443. We are using environment variables for configuration.

`APACHE_HTTP_PORT_NUMBER=8080`
`APACHE_HTTPS_PORT_NUMBER=8143`

The environment variables at set in the docker container:
`echo $APACHE_HTTP_PORT_NUMBER` returns `8080`
`echo $APACHE_HTTPS_PORT_NUMBER` returns `8143`

But Testlink is only reachable on port 80 and 443.

I also noticed that in the testlink-inputs.json in the docker container, the parameters for the Apache ports are missing. 

```# cat testlink-inputs.json
{
  "databaseName": "{{$global.env.TESTLINK_DATABASE_NAME}}",
  "databasePassword": "{{$global.env.TESTLINK_DATABASE_PASSWORD}}",
  "databaseServerHost": "{{$global.env.MARIADB_HOST}}",
  "databaseServerPort": "{{$global.env.MARIADB_PORT_NUMBER}}",
  "databaseUser": "{{$global.env.TESTLINK_DATABASE_USER}}",
  "email": "{{$global.env.TESTLINK_EMAIL}}",
  "language": "{{$global.env.TESTLINK_LANGUAGE}}",
  "password": "{{$global.env.TESTLINK_PASSWORD}}",
  "smtpConnectionMode": "{{$global.env.SMTP_CONNECTION_MODE}}",
  "smtpEnable": "{{$global.env.SMTP_ENABLE}}",
  "smtpHost": "{{$global.env.SMTP_HOST}}",
  "smtpPassword": "{{$global.env.SMTP_PASSWORD}}",
  "smtpPort": "{{$global.env.SMTP_PORT}}",
  "smtpUser": "{{$global.env.SMTP_USER}}",
  "username": "{{$global.env.TESTLINK_USERNAME}}"
}
```

**Steps to reproduce the issue:**

1. Setup a clean Testlink container
`curl -sSL https://raw.githubusercontent.com/bitnami/bitnami-docker-testlink/master/docker-compose.yml > docker-compose.yml`

2. Add the environment variables to the docker-compose.yml
```...
  testlink:
    image: 'bitnami/testlink:1.9.19'
    environment:
     ...
      - APACHE_HTTP_PORT_NUMBER=8080
      - APACHE_HTTPS_PORT_NUMBER=8143
    ports:
      - '80:8080'
      - '443:8143'
```

3. Start the docker container
docker-compose up

4. Open a webbrowser and go to http://localhost:80

**Describe the results you received:**

Testlink is not reachable.

**Describe the results you expected:**

Testlink should be reachable.

5. Change docker 

Changing the ports in the docker-compose.yml to

```
 testlink:
    image: 'bitnami/testlink:1.9.19'
    environment:
     ...
      - APACHE_HTTP_PORT_NUMBER=8080
      - APACHE_HTTPS_PORT_NUMBER=8143
    ports:
      - '80:80'
      - '443:443'
```

and restarting the container, makes testlink reachable on http://localhost:80.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

- Output of `docker version`:

```
Client: Docker Engine - Community
 Version:           19.03.5
 API version:       1.40
 Go version:        go1.12.12
 Git commit:        633a0ea838
 Built:             Wed Nov 13 07:29:52 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.5
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.12
  Git commit:       633a0ea838
  Built:            Wed Nov 13 07:28:22 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.10
  GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
 runc:
  Version:          1.0.0-rc8+dev
  GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683

```

- Output of `docker info`:

```
Client:
 Debug Mode: false

Server:
 Containers: 4
  Running: 2
  Paused: 0
  Stopped: 2
 Images: 43
 Server Version: 19.03.5
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: b34a5c8af56e510852c35414db4c1f4fa6172339
 runc version: 3e425f80a8c931f88e6d94a8c831b9d5aa481657
 init version: fec3683
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 4.15.0-76-generic
 Operating System: Ubuntu 18.04.3 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 17.43GiB
 Name: wessling-vm
 ID: 4FSI:37WH:6ACE:JCVK:NOGR:DGHU:GRIZ:FUEV:ZAB4:V7KM:IRK3:WUAI
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No swap limit support
```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.17.1, build unknown
docker-py version: 2.5.1
CPython version: 2.7.17
OpenSSL version: OpenSSL 1.1.1  11 Sep 2018
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

Running on different Linux distributions.

**Comment by miguelaeh** [comment]
Hi @Jacoon ,
Thank you very much for reporting this issue, we already took care of it and the image should be updated soon when it finish our testing process.
Regards.

**Comment by miguelaeh** [comment]
Hi @Jacoon ,
We found something weird while working on this, the last image that was released will not set the port either, we have to investigate a little deeper, we will update this thread when we have news.
Sorry for the noise.

**Comment by miguelaeh** [comment]
Hi again @Jacoon ,
Now the missing file into the `rootfs` folder is back again. You should be able to use the env var as you wanted.
The missing file was the `apache-inputs.json` that maps the env vars to variables into the initialization logic that is on Node.js.

**Comment by Jacoon** [comment]
Thank you @miguelaeh , for your fast help, I will test this as soon as possible. :)

**Comment by Jacoon** [comment]
Hi @miguelaeh, when will the update be available via docker hub? The last update of tag 1.9.19 was 22 days ago.

**Comment by miguelaeh** [comment]
Hi @Jacoon ,
Our images are updated every day, but it seems the latest version is the `1.9.20` so the `1.9.19` will not be updated anymore as it is already at his latest version. Since the `1.9.20` was released we will be updating that until the `1.9.21` will be released.
